src/journal_send.cc

```diff
@@ -1,14 +1,28 @@
 // Down below you will find the most intuitive code you probably ever saw on the
 // Internet. It just copies an ordinary JavaScript string array into an iovec
 // and then finally calls sd_journal_sendv. Rocket science included!
+// Priority is a special case as it needs to be taken from the syslog.h
+// preprocessor definitions, which aren't available from JavaScript.
 
 #include <nan.h>
 #include <systemd/sd-journal.h>
+#include <syslog.h>
+
+#define to_str_h(x) "" #x
+#define to_str(x) to_str_h(x)
+
+void setIovec( struct iovec *iov, const char *priorityStr ) {
+	size_t strLength = strlen(priorityStr) + 9;
+	iov->iov_len = strLength;
+	iov->iov_base = (char*) malloc( strLength + 1 );
+	snprintf((char*) iov->iov_base, strLength + 1, "PRIORITY=%s", priorityStr);
```

Comment:
Do you think we can utilse `%d` for the conversion of the priority integer to string? That way, we can just pass the integer if the log prio and do not need to convert all priority integers to strings.